### ISSUE TITLE
Tutorial to use scrcpy and sndcpy at same time

### ISSUE THREAD

**Issue by equiman** [rom1v/sndcpy#74]
First of all. I want to thank you to publish this project, it's amazing and works like a charm.

I've written an article to use both of them at the same time. Hope it helps.

https://dev.to/equiman/share-your-android-screen-and-audio-on-macos-1p64

**Comment by astromark** [comment]
Thanks. This inspired me to try the same with Linux. 

For anyone who is interested, first add the following to your .bashrc:
```
export SNDCPY_HOME=$HOME/sndcpy
export SNDCPY_APK=$HOME/sndcpy/sndcpy.apk
alias sndcpy="$HOME/sndcpy/sndcpy"
alias sndscr="$HOME/sndcpy/sndscr.sh"
```

Then create the sndscr.sh script:
```
#!/bin/bash

xterm -e sndcpy &
xterm -e scrcpy --disable-screensaver -f
```
Then make sure to make it executable with chmod +x.

Note that I have set the disable screensaver and fullscreen options for scrcpy as I'm using it for watching TV shows. You may or may not wish to include those.

**Comment by diazabdulm** [comment]
Is there a tutorial of some sort to do in Windows with Batch?

**Comment by diazabdulm** [comment]
Yeah, I resorted to creating a Batch script. It's rough, but it gets the job done. I also have it so an EXE shortcut once double-clicked runs the script. It's as seamless as possible without adding a GUI.

**Comment by SritharBoss** [comment]
> Yeah, I resorted to creating a Batch script. It's rough, but it gets the job done. I also have it so an EXE shortcut once double-clicked runs the script. It's as seamless as possible without adding a GUI.

Can you please share us the script?

**Comment by Lantern1984** [comment]
>> Yeah, I resorted to creating a Batch script. It's rough, but it gets the job done. I also have it so an EXE shortcut once double-clicked runs the script. It's as seamless as possible without adding a GUI.

> Can you please share us the script?

I'd like a copy too
